### PR TITLE
only LocalAppEngineConsole should be a console page participant

### LINKED ISSUE #567: MessageConsole ClassCastException when deploying
When I deploy, I see the following error. However, the console shows deploy output normally.

```
!MESSAGE Problems occurred when invoking code from plug-in: "org.eclipse.ui.console".
!STACK 0
java.lang.ClassCastException: org.eclipse.ui.console.MessageConsole cannot be cast to com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsole
	at com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant.init(LocalAppEngineConsolePageParticipant.java:31)
	at org.eclipse.ui.internal.console.ConsoleView$3.run(ConsoleView.java:329)
```

```
!ENTRY org.eclipse.ui.console 4 120 2016-08-16 11:59:48.082
!MESSAGE Error logged from Console plug-in: 
!STACK 0
java.lang.ClassCastException: org.eclipse.ui.console.MessageConsole cannot be cast to com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsole
	at com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant.init(LocalAppEngineConsolePageParticipant.java:31)
	at org.eclipse.ui.internal.console.ConsoleView$3.run(ConsoleView.java:329)
```

### PR DESCRIPTION
**Pull request by elharo**
close #567 #571

### CODE REVIEW

**Comment by elharo** [comment]
@chanseokoh @briandealwis

File: plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml [hunk 1]
```diff
@@ -139,6 +139,9 @@
       <consolePageParticipant
             class="com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant"
             id="com.google.cloud.tools.eclipse.appengine.consolePageParticipant">
+              <enablement>
+                <instanceof value="com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant"/>
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml]: This seems weird: check that the object is an instance of itself?  Should this be a `LocalAppEngineConsole`?

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml]: done

**Comment by briandealwis** [comment]
LGTM.  I'll commit this as I'm hitting it with other deploy testing.

**Pull request merged**